components/brave_rewards/core/state/state_keys.h

```diff
@@ -11,8 +11,6 @@ namespace state {
 
 const char kServerPublisherListStamp[] = "publisher_prefix_list_stamp";
 const char kUpholdAnonAddress[] = "uphold_anon_address";  // DEPRECATED
-const char kPromotionLastFetchStamp[] = "promotion_last_fetch_stamp";
```

Comment:
Should we rather just deprecate these?

Reply:
They are deprecated on the browser side. I don't think we need to keep around this string, do you?

Reply:
Right — no, that's fine. 👍🏻